lib/zipkin-tracer/config.rb

```diff
@@ -20,6 +21,9 @@ def initialize(app, config_hash)
       @json_api_host     = config[:json_api_host]
       # Zookeeper information
       @zookeeper         = config[:zookeeper]
+      # Kafka producer information
+      @kafka_producer    = config[:producer]
+      @kafka_topic       = config[:topic] if present?(config[:topic])
```

Comment:
True, for now. But that might change in the future. Someone can accidentally change it to an empty string, for instance. Since it's not a performance critical part, I prefer to have extra precautions. WDYT?